scalableminds/webknossos
# Reduce the free space between viewports in tracing

This PR adjusts the default width of a layout container of a viewport to its height so that there will be no more unused whitespace.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a tracing an reset the layout. There should be no more whitespace between the viewports.
- close all tracings, clear the local storage and then open a tracing. The default layout should not have any free space between the viewports.
- There is a maximum width for the viewports, so that the right menu is always visible
- Test this by resizing the window to a dimension where the height is greater than the width and the reset the layout. The right menu should be still visible.

### Issues:
- fixes #3298

------
